### ISSUE TITLE
[Enhancement] Damage command 

### ISSUE THREAD

**Issue by longdt1425** [promcteam/proskillapi#462]
**Is your feature request related to a problem? Please describe.**
Nope
**Describe the solution you'd like**
So i want a new command in plugin that deal damage to target entity without using skill's mechanic like: **_/class damage <caster> <target(use entity UUID or player's name)> <amount> <ignoreArmor:true/false> <preventknockback:true/false> <Classifier(for skill damage)[optional]> <Damage Cause>_**

**Describe alternatives you've considered**
.
**Additional context**
.

**Comment by Travja** [comment]
I'm a little confused why this is necessary? If you could give some context, that would be awesome. What exactly are you trying to accomplish?

**Comment by longdt1425** [comment]
> I'm a little confused why this is necessary? If you could give some context, that would be awesome. What exactly are you trying to accomplish?

Oh my bad, sorry for the lack of information. It's just that I'm trying to deal damage through mobs in MM but with skillapi mechanics. I know that the plugin already provides a command that allows mobs to use skillapi's skills but because MM has quite a lot of mechanics that it has so I can't directly use skillapi's skills. Then I have a suggestion to add a damage command and I also need the Damage Mechanic's Classifier from the skillapi to do some Took Skill Damage Condition related stuff. Sorry I use google translate.

**Comment by Travja** [comment]
And why can't you use the `/class mobcast` command?

**Comment by longdt1425** [comment]
> And why can't you use the `/class mobcast` command?

Yes I know that, but as mentioned above I already know this command exists and can't use it directly. If there is this command, it will help with something related to quests, events. It will also be very flexible.

**Comment by Travja** [comment]
Why can't you use it directly? Both solutions involve running a command

**Comment by longdt1425** [comment]
> Why can't you use it directly? Both solutions involve running a command

Since I use the mechanic from MM and I want to deal damage to the target from the MM, I can't use the target from Skillapi because it's not related. So if there is this command, it will be very convenient.

**Comment by lordsrubka5** [comment]
Are you sure? In what way does it not work? Does it not work on enemy target or on both enemy and ally

**Comment by longdt1425** [comment]
> Are you sure? In what way does it not work? Does it not work on enemy target or on both enemy and ally

Well, because I use both MM and SkillAPI plugins, but when I use MM's skills, it has a different target than the target from SkillAPI's system. So I wanted a command to deal damage to the MM's target but still use the correct SkillAPI damage mechanics since it has Classifier to do elemental damage, also use the Took Skill Damage Condition function to identify what type of damage was done received from Classifier.

**Comment by lordsrubka5** [comment]
So the targetters work, they are just not in a shape you want? What shape are you trying to pull off exactly, because it should be possible (or at least very similar)

**Comment by longdt1425** [comment]
> So the targetters work, they are just not in a shape you want? What shape are you trying to pull off exactly, because it should be possible (or at least very similar)

I can make the targeter work by some tricks like summoning mobs to the target and having that mob use mobcast command to deal damage, but the accuracy doesn't seem to be high so if there is such a command then everything is easier and reduce lag caused by too many mobs.
And also because I have to use some conditions from MM to activate the skill (from MM) and choose targets, so choosing the wrong target will not be okay.
